src/Nokia.xml

```diff
@@ -9,9 +9,7 @@
          <p>Version 1.0a</p>
       </titleText>
     
-      <list>
-        <item>
-            <bullet>1.</bullet>
+        <bullet>1.</bullet>
```

Comment:
`<bullet>` outside of an `<entry>` seems odd to me.  I thing we want to summport paragraphs in items (as HTML does) for cases like this where it's warranted.

An alternative here would be to use headers (and maybe allow `<bullet>` in them?) for these numbered sections.

Reply:
We do support paragraphs inside items - reference [XML schema] (https://github.com/spdx/license-list-XML/blob/19dcfd1b31c7e81582e5f23b91c0667544d4a678/schema/ListedLicense.xsd#L83)

We do not support anything other than items as children of a list tag - this is what caused the original parsing errors.

In looking at the source license text formatting, I decided that removing the top level list would be closer to the formatting than adding items around the bullet tags.

The semantics on the bullet tag is information that can be used for matching where an item is formatting hints used to indent properly (translates to li in HTML).  I think it should be fine to allow bullets outside of items.

We don't have any concept of headers in the current XML spec - perhaps something to be considered for a future release.

Reply:
> We don't have any concept of headers in the current XML spec - perhaps something to be considered for a future release.

I'd like to have a consistent way to address the numbered-(sub-)section use case, since many licenses do that.  Headers would be one way.  Using lists would be another, although it feels like you'd need a way to distinguish header-like list entries.  For example, currently [the GPL-2.0 does not mark up the “Preamble” header][1], although it's a header upstream:

```
$ curl -s https://www.gnu.org/licenses/old-licenses/gpl-2.0.html | grep Preamble
      <li><a id="TOC2" href="#SEC2">Preamble</a></li>
<h3 id="preamble"><a id="SEC2">Preamble</a></h3>
```

Our GPL-2.0 [does use lists to mark up the numbered sections][2], although [the “NO WARRANTY” header gets stuffed into the end of the previous `<item>`][3].

So I'm fine with whatever temporary workarounds you want to use, but I think we need to explicitly address the representation of upstream headers in the near future.

[1]: https://github.com/spdx/license-list-XML/blob/19dcfd1b31c7e81582e5f23b91c0667544d4a678/src/GPL-2.0.xml#L33
[2]: https://github.com/spdx/license-list-XML/blob/19dcfd1b31c7e81582e5f23b91c0667544d4a678/src/GPL-2.0.xml#L63-L65
[3]: https://github.com/spdx/license-list-XML/blob/19dcfd1b31c7e81582e5f23b91c0667544d4a678/src/GPL-2.0.xml#L251